output.go

```diff
@@ -0,0 +1,102 @@
+package plugin
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Output is the JSON payload that gets sent to Stderr from the plugin to the host
+type Output struct {
```

Comment:
Given this is exported (which seems fine), I would name this something more descriptive like `LogEntry`. 

`Output` is just too generic. :)